src/xero/manager.py

```diff
@@ -8,7 +8,7 @@ def __init__(self, name, credentials, unit_price_4dps=False, user_agent=None):
         from xero import __version__ as VERSION  # noqa
 
         self.credentials = credentials
-        self.name = name.capitalize()
+        self.name = name.capitalize() if name.islower() else name
```

Comment:
I had added this as a hack to make the tests behave "as expected".  But unfortunately that breaks the behaviour for all objects that have more than one upper case letter, e.g. "PurchaseOrders", as it enforce only the first letter is capitalized.

Now that I think about it, it should really be the tests that should be updated to use the CapitalizedNames of the objects rather than a lower case version...

Reply:
Agreed. The problem is with the test, not the manager.